### PR TITLE
add stateful order scripts

### PR DESCRIPTION
**Pull request by yang-dydx**
place one stateful order every N seconds.

### CODE REVIEW

File: place_stateful_orders.py [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+from v4_client_py.clients.helpers.chain_helpers import (
+    Order_TimeInForce,
+    calculate_side,
+    calculate_quantums,
+    calculate_subticks,
+    ORDER_FLAGS_LONG_TERM,
+    Order,
+)
+
+# The idea of this experiment is to see what is the lag between the remove and place messages when we replace orders
```
Comment (matthewdowney) [inline place_stateful_orders.py:33]: Nit: outdated I think

File: place_stateful_orders.py [hunk 2]
```diff
@@ -0,0 +1,252 @@
+"""Script that places stateful orders every N seconds
+   and writes the time that it was placed along with the order info to BigQuery
+
+Usage: python place_replacement_orders.py
```
Comment (matthewdowney) [inline place_stateful_orders.py:4]: Nit: Outdated I think

**Review by matthewdowney — APPROVED** [review]
Low priority item: pull out the duplicate code in the various place*.py scripts

**Comment by yang-dydx** [comment]
> Low priority item: pull out the duplicate code in the various place*.py scripts

will do this in a future PR

**Pull request merged**